lib/ostruct.rb

```diff
@@ -380,27 +380,59 @@ def delete_field(name, &block)
     end
   end
 
+  def self.__inspect_supports_identity_set?
```

Comment:
This is checked by the corresponding PR in `set`:

https://github.com/ruby/set/pull/33/files#diff-2dce49f3065d79cae2d369426180f3e0f28d484dbb22ea54e9cb70b6a87d3f4bR823